src/CartBlock.C

```diff
@@ -111,9 +111,15 @@ void CartBlock::getInterpolatedData(int *nints,int *nreals,int **intData,
 
 void CartBlock::update(double *qval, int index,int nq)
 {
+  if(index >= ncell_nf){
+    int stop = 0;
+    stop = stop -1;
```

Comment:
Looks like this is dead code?

Reply:
@sayerhs Thanks! I have fixed this in 136d8ba.